src/aea.f

```diff
@@ -1,45 +1,41 @@
 C> @file
-C
-C> ASCII TO EBCDIC, OR EBCDIC TO ASCII
-C> @author DESMARAIS @date 82-11-29
-C>
-C> CONVERT ASCII TO EBCDIC, OR EBCDIC TO ASCII BY CHARACTER.
-C> THIS SUBROUTINE CAN BE REPLACED BY CRAY UTILITY SUBROUTINES
-C> USCCTC AND USCCTT. SEE MANUAL SR-2079 PAGE 3-15. CRAY UTILITY TR
-C> CAN ALSO BE USED FOR ASCII, EBCDIC CONVERSION. SEE MANUAL SR-2079
-C> PAGE 9-35.
-C>
-C> PROGRAM HISTORY LOG:
-C> -  82-11-29  DESMARAIS
-C> -  88-03-31  R.E.JONES  CHANGE LOGIC SO IT WORKS LIKE A
-C>                        IBM370 TRANSLATE INSTRUCTION.
-C> -  88-08-22  R.E.JONES  CHANGES FOR MICROSOFT FORTRAN 4.10
-C> -  88-09-04  R.E.JONES  CHANGE TABLES TO 128 CHARACTER SET
-C> -  90-01-31  R.E.JONES  CONVERT TO CRAY CFT77 FORTRAN
-C>                        CRAY DOES NOT ALLOW CHAR*1 TO BE SET TO HEX
-C> -  98-12-21  Gilbert    Replaced Function ICHAR with mova2i.
-C>
-C>   INPUT ARGUMENT LIST:
-C>     IA       - CHARACTER*1 ARRAY OF ASCII DATA  IF NC < 0
-C>     IE       - CHARACTER*1 ARRAY OF EBCDIC DATA IF NC > 0
-C>     NC       - INTEGER,  CONTAINS CHARACTER COUNT TO CONVERT.
-C>                IF NC .LT. 0,  CONVERT ASCII TO EBCDIC
-C>                IF NC .GT. 0,  CONVERT EBCDIC TO ASCII
+C> @brief This subroutine converts ascii to ebcdic, or ebcdic to ascii
+C> @author desmarais @date 11-29-1982
+
+C> Program history log:
+C> @date 11-29-1982  @author desmarais
```

Comment:
We don't use the author tag in the Program history log section, or the date tag. See some of the other PRs for examples.